mkdocs.yml

```
@@ -5,13 +5,36 @@ repo_name: GitHub
 
 nav:
   - Home: index.md
-  - Contributing: CONTRIBUTING.md
-  - Code of Conduct: CODE_OF_CONDUCT.md
   - 🔗 Source code: https://github.com/DecodingRaphael/unraphael
   - 🔗 Issues: https://github.com/DecodingRaphael/unraphael/issues
+  - Background:
+    - Abstract: Background/ABSTRACT.md
+    - Data-analytical pipeline: Background/PIPELINE.md
+  - Getting Started:
+    - Installation: Started/INSTALLATION.md
+    - Tutorials: Started/TUTORIALS.md
+    - Known issues: Started/ISSUES.md
+  - Image Processing Steps:
+    - Introduction: Processing/INTRODUCTION.md
+    - Preprocessing: Processing/PREPROCESSING.md
+    - Segmentation: Processing/SEGMENTATION.md
+    - Alignment: Processing/ALIGNMENT.md
+    - Clustering: Processing/CLUSTERING.md
+    - Absolute area analysis: Processing/ANALYSIS.md
+  - Developers:
+    - Code of Conduct: CODE_OF_CONDUCT.md
+    - Building from source: Developers/SOURCE.md
+    - Contributing guidelines: CONTRIBUTING.md
+    - Changelog: Developers/CHANGELOG.md
+  - About:
+    - How to Cite: About/CITE.md
+    - Contact information: About/CONTACT.md
+    - In action: About/ACTION.md
+    - FAQ's: About/FAQS.md
 
 theme:
-  name: material
+  name: readthedocs
+  color_mode: dark
```

What are your reasons for changing the theme? The material theme is one of the best things about mkdocs imo.

```suggestion
  name: material
```

---

I changed it back again. The structure is, indeed, nice! Perhaps choose later a different color then purple?

---

I have no strong opinion on the color 😅